app/uris/parseZcashURI.ts

```diff
@@ -19,9 +20,16 @@ const parseZcashURI = async (uri: string): Promise<string | ZcashURITarget> => {
   // The first address is special, it can be the "host" part of the URI
   const address = parsedUri.pathname;
 
-  const resultParse = await RPCModule.execute('parse', address);
-  //console.log('parse', resultParse);
-  const resultParseJSON = await JSON.parse(resultParse);
+  const resultParse: string = await RPCModule.execute('parse', address);
+  if (resultParse) {
+    if (resultParse.toLowerCase().startsWith('error')) {
+      return 'Right now it is not possible to verify the address with the server';
```

Comment:
this error message is unclear

Reply:
I'm all ears, about some advice of the native English speakers...

Reply:
two days later, some idea about the perfect message here?

Reply:
@Oscar-Pepper can you help us with a more clear message?